certif/app/routes/authenticated/sessions/new.js

```diff
@@ -0,0 +1,18 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+  currentCertificationCenter: service(),
+
+  async model() {
+    const certificationCenter = await this.currentCertificationCenter.certificationCenter;
+
+    return this.get('store').createRecord('session', { certificationCenter });
```

Comment:
Pourquoi créer ici le record de "Session", avant même que l'utilisateur commence à le créer ?

Reply:
C'est the Ember way : tu créais (sans sauvegarder) ton objet dans le model() de ta route, qui est accessible par la suite dans ta vue, tu le remplie dans ton formulaire, et tu fais ton appel API au moment du click sur le bouton du formulaire

Reply:
Ember way (c'est fait aussi comme ça dans pix orga btw).